tutorials/movies/adding_a_movie_kind_field.md

```diff
@@ -100,17 +94,16 @@ If we build and run our project now, we'll see that there is no change in the Mo
 Modify *MovieForm.cs* as below:
 
 ```cs
-
 namespace MovieTutorial.MovieDB.Forms
 {
     // ...
     [FormScript("MovieDB.Movie")]
-    [BasedOnRow(typeof(Entities.MovieRow))]
+    [BasedOnRow(typeof(Entities.MovieRow), CheckNames = true)]
     public class MovieForm
     {
         // ...
-        public MovieKind Kind { get; set; }
         public Int32 Runtime { get; set; }
+        public MovieKind Kind { get; set; }
```

Comment:
is there a reason for switching places of Runtime and Kind in form?

Reply:
I thought it made more sense as we are currently adding Kind on MovieRow and it's below Runtime in the properties of the class